src/modal/test/modal.spec.js

```diff
@@ -833,6 +833,50 @@ describe('$uibModal', function () {
         expect($document.find('.modal-backdrop')).not.toHaveClass('fade');
       });
     });
+    
+    describe('container', function() {
+      it('should be added to body by default', function() {
+        var modal = open({template: '<div>Content</div>'});
+
+        expect($document).toHaveModalsOpen(1);
+        expect($document).toHaveModalOpenWithContent('Content', 'div');
+      });
+
+      it('should not be added to body if container is passed', function() {
+        var subElement = '<section>Some content</section>';
+        var element = angular.element(subElement);
```

Comment:
these two lines can be combined to one - there's not really any need to have them separate.